corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -44,7 +25,7 @@ function LocationSettingsViewModel() {
 
     this.new_loctype = function() {
         var parent_pk = (_.last(settings.loc_types()) || {}).pk;
-        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, this);
+        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, commtrack_enabled);
```

Comment:
I like the idea of flattening it out - just made that change and renamed/simplified a bunch of things.  I also switched from using a `load()` method to passing the data in in the constructor - it felt weird to pass in `commtrack_enabled` in the constructor, and `location_types` in a separate method.